system/hardware/hardwared.py

```diff
@@ -201,6 +201,10 @@ def hardware_thread(end_event, hw_queue) -> None:
   params = Params()
   power_monitor = PowerMonitoring()
 
+  uptime_offroad: float = params.get("UptimeOffroad", default=0.0)
+  uptime_onroad: float = params.get("UptimeOnroad", default=0.0)
```

Comment:
also I thought specifying everything in param_keys.h removed need for specifying at the getter

Reply:
? you specify the type in the param_keys.h, that was adeeb's point